lxd/storage/drivers/driver_alletra_wsapi1.go

```diff
@@ -483,3 +533,91 @@ func (p *alletraClient) deleteHost(hostName string) error {
 func (p *alletraClient) updateHost(hostName string, qns []string) error {
 	return fmt.Errorf("Failed to update host %q. Operation not supported.", hostName)
 }
+
+func (p *alletraClient) _createVolume(poolName string, volName string, sizeBytes int64) error {
```

Comment:
As discussed, when this is moved into the `clients` package, we can then differentiate between external and internal functions, by having an exported `CreateVolume()`  function and then renaming this to `createVolume()` for internal use.

Reply:
Yeah, I did it precisely like you've described.